File: app/src/main/java/com/absinthe/kage/media/video/VideoHelper.java
package com.absinthe.kage.media.video;

import android.graphics.Bitmap;
import android.media.ThumbnailUtils;
import android.provider.MediaStore;

import com.blankj.utilcode.util.FileUtils;

public class VideoHelper {

    public static Bitmap getVideoCoverImage(String filePath) {
        if (FileUtils.isFileExists(filePath)) {
            return ThumbnailUtils.createVideoThumbnail(filePath, MediaStore.Images.Thumbnails.MINI_KIND);
        }
        return null;
    }

}

File: app/src/main/kotlin/com/absinthe/kage/media/video/VideoHelper.kt
package com.absinthe.kage.media.video

import android.graphics.Bitmap
import android.media.ThumbnailUtils
import android.provider.MediaStore
import com.blankj.utilcode.util.FileUtils

object VideoHelper {
    fun getVideoCoverImage(filePath: String?): Bitmap? {
        return if (filePath != null && FileUtils.isFileExists(filePath)) {
            ThumbnailUtils.createVideoThumbnail(filePath, MediaStore.Images.Thumbnails.MINI_KIND)
        } else null
    }
}